### PR TITLE
[DOCS] Correct watcher event data example

### LINKED ISSUE #56248: Docs: watching-meetup-data.html example has 3 errors preventing example from working
Our meetup watcher example has 3 errors which prevent it from working as published

doc page:
https://www.elastic.co/guide/en/elasticsearch/reference/master/watching-meetup-data.html

errors:
(1)
        "indices": [
          "<logstash-{now-1h}>",
          "<logstash-{now}>"
        ],
doesn't work with the index names logstash will create if ILM is enabled, e.g.
logstash-2020.05.05-000001

I believe these indices should have "-*" appended, but please verify this for exactly what values are correct under the default ILM policy "logstash-policy"

(2) the field for the three aggs in the query should reference .keyword and not .raw, otherwise the aggs don't work

         "aggs": {
            "group_by_city": {
              "terms": {
                "field": "group.group_city.raw",  <-- should be .keyword
                "size": 5
              },
              "aggs": {
                "group_by_event": {
                  "terms": {
                    "field": "event.event_url.raw",  <-- should be .keyword
                    "size": 5
                  },
                  "aggs": {
                    "get_latest": {
                      "terms": {
                        "field": "@timestamp", 
                        "size": 1,
                        "order": {
                          "_key": "desc"
                        }
                      },
                      "aggs": {
                        "group_by_event_name": {
                          "terms": {
                            "field": "event.event_name.raw"  <-- should be .keyword
                          }
                        }
                      }
                    }
                  }

(3) condition should be ctx.payload.hits.total and not ctx.payload.hits.total.value -- the latter does not exist in the result body and the condition is never met

INCORRECT:
  "condition": {
    "compare": {
      "ctx.payload.hits.total.value": {
        "gt": 0
      }
    }
  },

CORRECT:
  "condition": {
    "compare": {
      "ctx.payload.hits.total": {
        "gt": 0
      }
    }
  },

### PR DESCRIPTION
**Pull request by jrodewig**
* Updates Logstash index patterns. Logstash indices now default to an
  ILM-friendly index pattern.
* Swaps `*.raw` fields for `*.keyword` fields. The Logstash template
  uses `keyword` fields by default since 6.x.
* Swaps instances of`ctx.payload.hits.total.value` with
  `ctx.payload.hits.total`

Closes #56248

### CODE REVIEW

**Comment by hub-cap** [comment]
@jrodewig its best if someone from @elastic/es-core-features reviews this since im no longer on the team working on watcher.

**Comment by jrodewig** [comment]
@hub-cap Will do. Thanks for the heads up!

File: x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc [hunk 1]
```diff
@@ -83,8 +83,8 @@ To set up the watch:
     "search": {
       "request": {
         "indices": [
-          "<logstash-{now-1h}>", <1>
-          "<logstash-{now}>"
```
Comment (jakelandis) [inline x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc:87]: With ILM there will be an alias by default named `logstash`. Rather then trying to query the concrete indices, I think the example should just use a single index `logstash` and the note below call out that `logstash` is an alias. 

In general we should avoid examples that include both time based (e.g. {now}) and rollover based (e.g. -00001) in favor of the fronting alias. The time stamp on the index when combined with rollover changes meaning such that it is the time when the index was created, not necessarily reflective of the content of that index (the content is dependent on the rollover rules). It is a distinction difficult to describe and the alias should be the preferred access across multiple rolled over indices.

Comment (jrodewig) [reply x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc:87]: Thanks @jakelandis. I've updated this and the following snippet to use the `logstash` index alias with 2c1d2db.

Also added some more context around Logstash ILM defaults in the callout text.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Review by jakelandis — APPROVED** [review]
LGTM - thanks for fixing this !

**Comment by jrodewig** [comment]
### Backport commits
master 1353d8b
7.x cf76a93 
7.8 3661f64
7.7 4b578c3 
7.6 9e03d03

**Pull request merged**